### DISCUSSION TITLE
Where did Luke Skywalker first meet Han Solo in the original Star Wars film?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#98] [Galaxy Trivia] [answered]
**Hint:** 🍹

**Comment by Tynab** [comment, marked as answer]
Luke Skywalker first met Han Solo in the original Star Wars film (later retitled "Star Wars: Episode IV - A New Hope") at the Mos Eisley Cantina on the planet Tatooine. Specifically, they met when Han Solo saved Luke from a group of hostile aliens who were about to attack him. Han agreed to take Luke and his droids, R2-D2 and C-3PO, on his ship, the Millennium Falcon, for a fee. This marked the beginning of their adventure together.
